BranchMetrics/ios-branch-deep-linking-attribution
# SDK-1329 SDK layout update

## Reference
SDK-1329 SDK layout update

## Summary
The Branch SDK has a usual project layout which causes a large number of warnings when using SPM. The root cause is the Branch header is being used as the framework umbrella header and a class header.

There are a few more minor issues also being addressed with this change.

## Type Of Change

## Testing Instructions
Test integration options.
1. SPM - umbrella header warnings should be gone.
2. Cocoapods
3. Carthage
4. Manual xcframework

Confirm TestBed unit tests still work.

Confirm development is not hindered by the changes.









cc @BranchMetrics/saas-sdk-devs for visibility.
